src/index.html

```diff
@@ -1,14 +1,87 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible"
+          content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect"
+          href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+          rel="stylesheet">
+    <link rel="stylesheet"
+          href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header-container container">
+        <a href="#"
+           class="header-logo">
+          <img src="./images/logo.png"
+               alt="logo">
+        </a>
+        <nav class="header-menu menu">
+          <ul class="menu-list">
+            <li class="menu-item">
+              <a href="#"
+                 class="menu-link is-active">
+                Apple
+              </a>
+            </li>
+            <li class="menu-item">
+              <a href="#"
+                 class="menu-link">
+                Samsung
+              </a>
+            </li>
+            <li class="menu-item">
+              <a href="#"
+                 class="menu-link">
+                Smartphones
+              </a>
+            </li>
+            <li class="menu-item ">
+              <a href="#"
+                 class="menu-link "
+                 data-qa="hover">
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="menu-item">
+              <a href="#"
+                 class="menu-link">
+                Gadgets
+              </a>
```

Comment:
```suggestion
              <a href="#" class="menu-link">
                Gadgets
              </a>
```
if you have only two attributes you can write it in one line